spec/lucky_cli/runner_spec.cr

```diff
@@ -21,6 +21,30 @@ describe LuckyCli::Runner do
       .should have_called_my_cool_task
   end
 
+  it "prints the help_message for a found task when a help flag is passed" do
+    LuckyCli::Runner
+      .run(args: ["my.cool_task", "--help"])
+      .should have_default_help_message
+    LuckyCli::Runner
+      .run(args: ["my.cool_task", "-h"])
+      .should have_default_help_message
+    LuckyCli::Runner
+      .run(args: ["my.cool_task", "help"])
+      .should have_default_help_message
+  end
+
+  it "prints a custom help_message when set" do
+    LuckyCli::Runner
+      .run(args: ["my.custom_name", "-h"])
+      .should eq "Run with 'lucky my.custom_name'"
+  end
+
+  it "still calls a found task with other options passed" do
```

Comment:
Rather than "with other options passed" I think it would be extra clear if it were something like `with non help options`. What do you think?